src/items/class-features/inspiring_boost_(ex).json

```diff
@@ -37,7 +37,7 @@
       "gmnotes": "",
       "short": "",
       "unidentified": "",
-      "value": "<p>&gt;@UUID[Compendium.sfrpg.class-features.hIhgdXnqGHBm0A64]{Envoy Improvisation}</p>\n<p>&nbsp;@Icon[type:language-dependent] Language-Dependent</p>\n<p>&nbsp;@Icon[type:mind-affecting] Mind-Affecting</p>\n<p>@Icon[type:sense-dependent] Sense-Dependent</p>\n<p>As a standard action, you can signal an ally within 30 feet who has taken damage from any attack made by a significant enemy (see page 242) at any point after your last turn ended. That ally regains a number of Stamina Points (up to his maximum) equal to twice your envoy level + your Charisma modifier; at 15th level, this increases to three times your envoy level + your Charisma modifier. Once an ally has benefited from your inspiring boost, that ally can’t gain the benefits of your inspiring boost again until he takes a 10-minute rest to recover Stamina Points.</p>\n<p>At 6th level, you can spend 1 Resolve Point to add your envoy level to the number of Stamina Points regained.</p>"
+      "value": "<p>&gt;@UUID[Compendium.sfrpg.class-features.hIhgdXnqGHBm0A64]{Envoy Improvisation}</p>\n<p>As a standard action, you can signal an ally within 30 feet who has taken damage from any attack made by a significant enemy (see page 242) at any point after your last turn ended. That ally regains a number of Stamina Points (up to his maximum) equal to [[@classes.envoy.levels*2+@abilities.cha.mod]]{twice your envoy level + your Charisma modifier}; at 15th level, this increases to [[@classes.envoy.levels*3+@abilities.cha.mod]]{three times your envoy level + your Charisma modifier}. Once an ally has benefited from your inspiring boost, that ally can’t gain the benefits of your inspiring boost again until he takes a 10-minute rest to recover Stamina Points.</p>\n<p>At 6th level, you can spend 1 Resolve Point to add your envoy level to the number of Stamina Points regained.</p>"
```

Comment:
You've also managed to lose the added `@Icon` text from a recent commit, and the descriptors are being set back to false, and source reverted

Reply:
No clue why my VSCode reset my entries....Hopefully fixed now